hildon/hildon-entry.c

```diff
@@ -67,11 +67,15 @@ set_property                                    (GObject      *object,
     switch (prop_id)
     {
     case PROP_SIZE:
-	size = g_value_get_flags (value);
-	/* If this is auto height, default to finger height. */
-	if (!(size & (HILDON_SIZE_FINGER_HEIGHT | HILDON_SIZE_THUMB_HEIGHT)))
-	  size |= HILDON_SIZE_FINGER_HEIGHT;
-	hildon_gtk_widget_set_theme_size (GTK_WIDGET (object), size);
+        size = g_value_get_flags (value);
+        /* If this is auto height, default to finger height. */
+        if (!(size & (HILDON_SIZE_FINGER_HEIGHT | HILDON_SIZE_THUMB_HEIGHT)))
+            size |= HILDON_SIZE_FINGER_HEIGHT;
+#ifdef MAEMO_GTK
+        hildon_gtk_widget_set_theme_size (GTK_WIDGET (object), size);
```

Comment:
This line should remain unchanged in this commit.
Please, add it to style-fixing commit mentioned above.